### ISSUE TITLE
MISP better configurability

### ISSUE THREAD

**Issue by BenjaminH-Sogeti** [intelowlproject/IntelOwl#967]
# Issue
MISP has actually a really low level of configurability
- Hardcoded time constraint (events not older than 90 days)
- Hardcoded results limitation (50)
- Type of attribute filter is mandatory
- Search is only made on events field and not on attributes

# Doc 
Search() method start [here](https://buildmedia.readthedocs.org/media/pdf/pymisp/latest/pymisp.pdf) at page 29 

# Improvements
Offer a better configurability by moving these value into analyzer_config.json params and make them optional in case of null value.

**Comment by mlodic** [comment]
yeah, thanks for your interest, but policy wants that each person should just have one assigned task at a time.

**Comment by BenjaminH-Sogeti** [comment]
After our discussion in #979 
And discovering the different types of misp [data models](https://www.misp-project.org/datamodels/), I think that it could be useful to add "generic ioc" search format without type-filter. 

It could be useful to search for IBAN, filename, ... (everything that could be set into misp)

https://github.com/intelowlproject/IntelOwl/blob/ceebcc5042885ba49af564106a6cf419f5a31f63/configuration/analyzer_config.json#L1561

I may handle it if it also make sense to you !

**Comment by mlodic** [comment]
yeah it makes sense. Consider that it was not there before because we added the "generic ioc" feature later than that MISP analyzer. Would you like to make a PR for that? In case, I can wait for that to make a new release with everything you already made. Thank you!

**Comment by BenjaminH-Sogeti** [comment]
> Would you like to make a PR for that?

Sure
